Note re exception from storage.managed.get (#36586)
mdn/content: files/en-us/mozilla/add-ons/webextensions/api/storage/storagearea/get/index.md

### Return value

> A [`Promise`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) that resolves to a `results` object, containing every object in `keys` found in the storage area. If `keys` is an object, keys that are not found in the storage area have their values given by the `keys` object.

Author: Somewhere between the two
```suggestion
A [`Promise`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) that resolves to a `results` object, containing every object in `keys` found in the storage area. When `keys` is an object, any key that isn't found in storage takes its value from the `keys` object.
```

If the operation fails, the promise is rejected with an error message.

If managed storage is not set, `undefined` is returned.

> [!WARNING]

> > In Firefox, if the managed storage manifest is missing, an exception is thrown when using this function to access managed storage (see [Firefox bug 1868153](https://bugzil.la/1868153)). This issue can be avoided by catching the error. This issue is related to the lack of support for the [`storage.managed_schema`](/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/storage) manifest key (see [Firefox bug 1771731](https://bugzil.la/1771731)). `storage.managed_schema` provides a fallback managed storage manifest definition on other browsers.

Reviewer: This isn't quite right. Firefox does not currently support the `storage.managed_schema` manifest key, so it's presence or absence doesn't matter. In other browsers the JSON file specified in this key doesn't provide fallbacks, it defines the shape of the data that can be exposed via `storage.managed`. 

The core issue is that Firefox only uses the [`3rdparty` policy](https://mozilla.github.io/policy-templates/#3rdparty) to configure the managed storage area for an extension. If Firefox cannot load the configuration settings for a given extension from this policy, attempting to read from it will generate a runtime error. 

I took a shot at revising this paragraph for clarity.

```suggestion
> In Firefox, if an extension's managed storage has not been configured via the [`3rdparty` enterprise policy](https://mozilla.github.io/policy-templates/#3rdparty), an exception is thrown when using this function to access managed storage (see [Firefox bug 1868153](https://bugzil.la/1868153)). This issue can be avoided by catching the error. This issue is related to the lack of support for the [`storage.managed_schema`](/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/storage) manifest key (see [Firefox bug 1771731](https://bugzil.la/1771731)).
```
